api/src/wfl/module/covid.clj

```diff
@@ -179,6 +258,12 @@
                     {:id       items
                      :workload workload}))))
 
+(defoverload workloads/create-workload! pipeline create-covid-workload)
+(defoverload workloads/start-workload! pipeline start-covid-workload)
+(defoverload workloads/update-workload! pipeline update-covid-workload)
+(defoverload workloads/stop-workload! pipeline batch/stop-workload!)
+(defoverload workloads/load-workload-impl pipeline load-covid-workload-impl)
```

Comment:
Perhaps left over from a previous merge, but all these (and others) are defined line 665

Reply:
Removed